tianshou/policy/modelfree/ddpg.py

```diff
@@ -131,16 +133,15 @@ def forward(self, batch: Batch,
             Please refer to :meth:`~tianshou.policy.BasePolicy.forward` for
             more detailed explanation.
         """
-        model = getattr(self, model)
         obs = getattr(batch, input)
-        actions, h = model(obs, state=state, info=batch.info)
+        actions, h = getattr(self, model)(obs, state=state, info=batch.info)
```

Comment:
why changing this ? I don't remember the rational behind `getattr(self, model)` vs `self.model`

Reply:
Hmm, this is another mypy error. 

First, `getattr(self, model)` is not equal to self.model (self.model is `getattr(self, 'model')`;

Second, mypy doesn't accept a variable to be assigned with a different type. In this scenario, `model` is a string previously (as indicated in the argument list), however, `model` is re-assigned to be a `torch.nn.Module` in `model = getattr(self, model)`. This will raise an error in mypy:
```
tianshou/policy/modelfree/dqn.py: note: In member "forward" of class "DQNPolicy":
tianshou/policy/modelfree/dqn.py:135: error: "str" not callable  [operator]
            q, h = model(obs_, state=state, info=batch.info)
                   ^
Found 1 error in 1 file (checked 47 source files)
```

Reply:
I don't like the idea to make everything more complex and difficult to understand just because the type checker is shit. It is a good idea in principle but currently It feels like it is just encouraging but practice.

Reply:
I’ll change this to a better way